### DISCUSSION TITLE
Export Files in "Layout"

### DISCUSSION THREAD

**Discussion by kevin20252025** [djyt/cannonball#164] [Q&A]
Hello, my question is: how can I export or convert my project from the "Layout" program to create an executable (".exe" or "rom") so I can share it with my friend and let them play with my edited tracks?  

When I use the option "file" -> "export" -> "canoball," it only generates a ".bin" file, and I don’t know how to run it.  

Thanks in advance (my native language is Spanish).  

![Captura de pantalla (427)](https://github.com/user-attachments/assets/8c267670-ddf3-4126-a1c5-743c37f4ada4)

**Comment by Beastman1976** [comment]
Create a shortcut to cannonball on your desktop, go to properties, and for 'target' you need to add " -file WhateverYourLayoutFileIsCalled.bin" after "cannonball.exe"  

Or tell your friend to install Layout and send him your project .xml file with it to run from Layout.
